src/test/com/cw/mapper/GalacticInputMapperTest.java

```diff
@@ -0,0 +1,92 @@
+package com.cw.mapper;
+
+import main.com.cw.domain.GalacticCode;
+import main.com.cw.domain.GalacticQuery;
+import main.com.cw.domain.Metal;
+import main.com.cw.mapper.GalacticInputMapper;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by cwijayasundara on 23/01/2017.
+ */
+
+public class GalacticInputMapperTest {
+
+    private GalacticInputMapper galacticInputMapper;
+
+    @Before
+    public void setUp() throws Exception {
+        galacticInputMapper = new GalacticInputMapper();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+
+    }
+
+    @Test
+    public void shoudTestInputtoGalacticCodeMapperWithSampleValues(){
+        List<GalacticCode> tokenToRomanMap = galacticInputMapper.mapInputToGalacticCodes();
```

Comment:
I cannot see where these values fed in to the test.. (am i missing something?) Or is this solution hard bounded to the values asserted below?